WordPressAuthenticator/Unified Auth/View Related/Get Started/GetStartedViewController.swift

```diff
@@ -734,3 +830,30 @@ extension GetStartedViewController: UITextFieldDelegate {
     }
 
 }
+
+// MARK: - Keyboard Notifications
+
+extension GetStartedViewController {
+    @objc func handleKeyboardWillShow(_ notification: Foundation.Notification) {
+        keyboardWillShow(notification)
+    }
+
+    @objc func handleKeyboardWillHide(_ notification: Foundation.Notification) {
+        keyboardWillHide(notification)
+    }
+}
+
+// MARK: - Button configuration
+
+private extension GetStartedViewController {
+    enum ButtonConfiguration {
+        enum Continue {
+            static let title = WordPressAuthenticator.shared.displayStrings.continueButtonTitle
+            static let accessibilityIdentifier = "Get Started Email Continue Button"
+        }
+
+        enum SignInWithSiteCredentials {
+            static let accessibilityIdentifier = "Sign in with site credentials Button"
```

Comment:
Similar suggestion as above:
```suggestion
            static let accessibilityIdentifier = "sign-in-with-site-credentials-button"
```

Reply:
To keep it consistent with [the other style](https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/649#discussion_r929609931) I would like to keep the existing style.